app/src/main/java/com/nononsenseapps/feeder/ui/compose/deletefeed/DeleteFeedScreen.kt

```diff
@@ -86,14 +103,44 @@ fun DeleteFeedDialog(
             }
         },
         title = {
-            Text(
-                text = stringResource(id = R.string.delete_feed),
-                style = MaterialTheme.typography.titleLarge,
-                textAlign = TextAlign.Center,
-                modifier =
-                    Modifier
-                        .padding(vertical = 8.dp),
-            )
+            Box(modifier = Modifier.fillMaxWidth()) {
+                val stateLabel = if (selectAllSelected) {
+                    stringResource(androidx.compose.ui.R.string.selected)
+                } else {
+                    stringResource(androidx.compose.ui.R.string.not_selected)
+                }
+                Text(
+                    text = stringResource(id = R.string.delete_feed),
+                    style = MaterialTheme.typography.titleMedium,
+                    modifier = Modifier.align(Alignment.CenterStart)
+                )
+                Box(modifier = Modifier.align(Alignment.CenterEnd)) {
+                    IconButton(onClick = { selectAllButton = true }) {
+                        Icon(Icons.Default.MoreVert, contentDescription = "Select All")
+                    }
+
+                    DropdownMenu(expanded = selectAllButton, onDismissRequest = { selectAllButton = false }) {
+                        DropdownMenuItem(text = {
+                            Row(verticalAlignment = Alignment.CenterVertically) {
+                                Checkbox(checked = selectAllSelected, onCheckedChange = { checked ->
```

Comment:
IMO this checkbox can be removed. It's just extra logic to keep that checkbox in a correct state. What do you think?

Reply:
What do you prefer instead of checkbox? A normal button?

Reply:
no just let it be text in that case. most dropdowns in android are just plain text

Reply:
Alright will update

Reply:
Remove the checkbox, But I personally would prefer it tho, It would give me more insights if all the values are selected or not